soh/soh/Enhancements/randomizer/logic.cpp

```diff
@@ -441,9 +441,12 @@ namespace Rando {
 
     uint8_t Logic::BottleCount() {
         uint8_t count = 0;
+        if (!CanEmptyBigPoes){
```

Comment:
Why was this added here? It shouldn't return nothing if there are multiple bottles, as only one is included in the pool, and even then not all the time.